recipe/meta.yaml

```diff
@@ -54,7 +36,50 @@ outputs:
         - clangdev {{ version }}|{{ version }}.*
         - llvmdev {{ version }}|{{ version }}.*
         - openmp {{ version }}|{{ version }}.*
+    about:
+      home: http://openmp.llvm.org/
+      license: NCSA
+      license_file: LICENSE.txt
+      summary: The OpenMP API supports multi-platform shared-memory parallel programming in C/C++ and Fortran.
+    test:
+      requires:
+        - clangdev
+        - {{ compiler('cxx') }}  # [unix]
+        - vs2017_{{ target_platform }}  # [win]
+      commands:
+        - if not exist %LIBRARY_BIN%\\libomp.dll exit 1   # [win]
+        - test -f $PREFIX/lib/libomp.so                   # [linux]
+        - test -f $PREFIX/lib/libomp.dylib                # [osx]
+        - if not exist %LIBRARY_LIB%\\libomp.lib exit 1   # [win]
+        - if not exist %LIBRARY_INC%\\omp.h exit 1        # [win]
+        - test -f $PREFIX/include/omp.h                   # [unix]
+        - $PREFIX/bin/clang -v -fopenmp -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib omp_hello.c -o omp_hello  # [not win]
+        - "%LIBRARY_BIN%\\clang -v -fopenmp -I%LIBRARY_INC% -L%LIBRARY_LIB% omp_hello.c -o omp_hello.exe"  # [win]
+        - ./omp_hello            # [unix]
+        - "%cd%\\omp_hello.exe"  # [win]
+      files:
+        - omp_hello.c
+
+  - name: openmp_impl                                        # [linux]
+    script: build-openmp_impl.sh                             # [linux]
+    version: {{ openmp_ver }}                                # [linux]
+    requirements:                                            # [linux]
+      run:                                                   # [linux]
+        - llvm-openmp                                        # [linux]
```

Comment:
If you add `llvm-openmp` to `host`, then you don't need the post-link hack.

Reply:
Hmmmm. I guess the host deps are not uninstalled when conda is packaging the files? I just pushed a fix to conda build to allow symlinks to nowhere as well. Trying this now.

Thanks!